modules/sdk/gradle-plugins-workspace/src/gradleTest/useLatestNode/build.gradle

```diff
@@ -0,0 +1,17 @@
+task runGradleTest
+
+node {
+	useLatestNode = "true"
+}
+
+runGradleTest {
+	dependsOn downloadNode
+
+	doLast {
+		assert node.nodeVersion != null
+		assert node.nodeVersion != "16.13.0"
```

Comment:
@simonjhy this will break whenever Node's LTS version updates, right? I think this needs to pull in the live JSON if possible.

Reply:
this will not break test when node list update because the 16.13.0 is default node version setting by gradle-plugins. We just make sure the node version is not same the default then the test should be ok

Reply:
@drewbrokke do you mean we need to download the node version json file to verify the node version we configured is correct?

Reply:
@simonjhy I see, I commented on the l. I mean to comment this on the line below, but I think that will work fine for now since it will probably be a while before the version numbers go above the twenties.

Reply:
@drewbrokke I remove the Improper line assert code.